\documentclass[10pt]{beamer}
\usetheme{Combo}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}
\usepackage{graphicx}
\usepackage{epstopdf}
\usepackage{tikz}
\usepackage{amsmath}
\usepackage{fixltx2e}
\usetikzlibrary{shapes,arrows,positioning}
\author{Giovanni Bacci}
\title{Mining Microbiomes}
\subtitle{Computational Biology approaches to uncover the complexity of bacterial communities}
\institute{University of Florence\\
CRA-RPS}
\date{February 10, 2015}
%\setbeamercovered{transparent} 
%\setbeamertemplate{navigation symbols}{} 
%\logo{} 
%\date{} 
%\subject{} 
\begin{document}

\begin{frame}[plain]
\titlepage
\end{frame}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - BACKGROUND
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Background}
\subsection{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 1 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
\vspace{4mm}
\begin{center}
	{\Large\textbf{A bacterial World...}}
\end{center}
\vspace{-2mm}
		\begin{center}
			\begin{overlayarea}{1\textwidth}{0.2\textheight}
			\only<1>{
			\begin{block}{}
				\begin{center}
					Bacteria are often cited as examples of one of the Earthâ€™s most primitive living forms		
				\end{center}
			\end{block}
			}
			\only<2>{
			\begin{block}{}
				\begin{center}
					They have been always considered from an anthropocentric perspective
				\end{center}
			\end{block}
			}
			\only<3>{
			\begin{block}{}
				\begin{center}
					They are still associated almost exclusively to infection diseases
				\end{center}
			\end{block}
			}
			\only<4>{
			\begin{block}{}
				\begin{center}
					They are often studied to develop new antibiotic treatments
				\end{center}
			\end{block}
			}
			\only<5>{
			\begin{block}{}
				\begin{center}
					But reality is rather different
				\end{center}
			\end{block}
			}
			\only<6>{
			\begin{block}{}
				\begin{center}
					Only a very small fraction of bacteria is able to cause infections
				\end{center}
			\end{block}
			}
			\only<7>{
			\begin{block}{}
				\begin{center}
					A highly diversified and beneficial bacterial world exists
				\end{center}
			\end{block}
			}
			\end{overlayarea}
		\end{center}
		\begin{center}
			\begin{overlayarea}{1\textwidth}{0.6\textheight}
			\only<1>{\begin{center}\includegraphics[width=0.8\textwidth]{./images/bacwcartoon/bacterial_world1-01}\end{center}}
			\only<2>{\begin{center}\includegraphics[width=0.8\textwidth]{./images/bacwcartoon/bacterial_world2-01}\end{center}}
			\only<3>{\begin{center}\includegraphics[width=0.8\textwidth]{./images/bacwcartoon/bacterial_world3-01}\end{center}}
			\only<4>{\begin{center}\includegraphics[width=0.8\textwidth]{./images/bacwcartoon/bacterial_world4-01}\end{center}}
			\only<5>{\begin{center}\includegraphics[width=0.8\textwidth]{./images/bacwcartoon/bacterial_world5-01}\end{center}}
			\only<6>{\begin{center}\includegraphics[width=0.8\textwidth]{./images/bacwcartoon/bacterial_world6-01}\end{center}}
			\only<7>{\begin{center}\includegraphics[width=0.8\textwidth]{./images/bacwcartoon/bacterial_world7-01}\end{center}}	
			\end{overlayarea}		
		\end{center}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 1 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 2
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\begin{overlayarea}{1\textwidth}{0.9\textheight}
	\only<1>{\begin{center}\includegraphics[width=0.7\textwidth]{./images/diversity/div-1}\end{center}}
	\only<2>{\begin{center}\includegraphics[width=0.7\textwidth]{./images/diversity/div-2}\end{center}}
	\only<3->{\begin{center}\includegraphics[width=0.7\textwidth]{./images/diversity/div-3}\end{center}}
	\vspace{-2mm}
	\only<4->{
	\begin{block}{}	
	\begin{center}
		\textit{Metagenomics (also referred to as environmental and community genomics) is the genomic analysis of microorganisms by direct extraction and cloning of DNA from an assemblage of microorganisms.}\\
		\hfill\cite{handelsman2004metagenomics}
	\end{center}		
	\end{block}
	}
	\end{overlayarea}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 2
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 3
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\begin{columns}
		
		\begin{column}{0.5\textwidth}
			\vspace{-2mm}
			\begin{center}
				\Large\textbf{Great plate count anomaly}	
			\end{center}
			\begin{center}			
			\includegraphics[width=\textwidth]{./images/Great-plate-count-anomaly}
			\end{center}
		\end{column}
		
		\begin{column}{0.5\textwidth}
			\begin{overlayarea}{0.9\textwidth}{0.9\textheight}
			\vspace{1mm}				
			\small{%
				\only<1->{%
				\begin{center}
					Counts of cells obtained via cultivation are orders of magnitude lower than those directly observed via microscope \cite{staley1985measurement}
				\end{center}}%
				\only<2->{%
				\begin{alertblock}{}				
				\begin{center}
					Using standard laboratory techniques, microbiologists are able to cultivate only 1\% of existent bacteria \cite{hugenholtz2002exploring}				
				\end{center}					
				\end{alertblock}}%				
			}%
			\begin{center}			
				\only<3->{\includegraphics[width=0.9\textwidth]{./images/metagenomics}}
			\end{center}
			\end{overlayarea}
		\end{column}
		
	\end{columns}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 3
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START- Slide 4
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\begin{overlayarea}{\textwidth}{0.95\textheight}
		\only<1>{%
			\begin{center}
				\includegraphics[width=0.93\textwidth]{./images/uber_pipeline_01}
			\end{center}}%
		\only<2>{%
			\begin{center}
				\includegraphics[width=0.93\textwidth]{./images/uber_pipeline_02}
				\vspace{-5cm}
				\begin{columns}
					\begin{column}{0.3\textwidth}
					% Nothing here
					\end{column}
					\begin{column}{0.3\textwidth}
						\begin{center}						
						\begin{alertblock}{}
						\begin{center}
							All approaches are based on high throughput sequencing technologies					
						\end{center}					
						\end{alertblock}
						\end{center}
					\end{column}	
					\begin{column}{0.4\textwidth}
					% Nothing here
					\end{column}
				\end{columns}			
			\end{center}}%	
	\end{overlayarea}		
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 4
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 5
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\vspace{-6mm}
	\begin{overlayarea}{\textwidth}{\textheight}
	\begin{block}{DNA sequencing timeline}
	\begin{center}
	%\large\textbf{{DNA sequencing timeline}}
	\scriptsize{
	\begin{tikzpicture}
	
		% Vertical lines (all the same)
		%\foreach \x in {0, 3.87, 4.30, 8.17, 9.03, 9.89}
		%\draw [darkgray, thick] (\x cm,3pt) -- (\x cm,-3pt);
		
		% Vertical lines all different
		% 1977 - 0x down
		\draw [darkgray, thick] (0,0) -- (0cm,-8pt);
		\draw [darkgray, fill] (0cm, -8pt) circle [radius=0.04];
		
		% 1986 - 3.87x up
		\draw [darkgray, thick] (3.87,0) -- (3.87cm, 26pt);
		\draw [darkgray, fill] (3.87cm, 26pt) circle [radius=0.04];
		
		% 1987 - 4.30x down
		\draw [darkgray, thick] (4.30,0) -- (4.30cm,-8pt);
		\draw [darkgray, fill] (4.30cm, -8pt) circle [radius=0.04];
		
		% 1996 - 8.17x up
		\draw [darkgray, thick] (8.17,0) -- (8.17cm, 26pt);
		\draw [darkgray, fill] (8.17cm, 26pt) circle [radius=0.04];
		
		% 1998 - 9.03x down
		\draw [darkgray, thick] (9.03,0) -- (9.03cm,-8pt);
		\draw [darkgray, fill] (9.03cm, -8pt) circle [radius=0.04];
		
		% 2000 - 9.89x up
		\draw [darkgray, thick] (9.89,0) -- (9.89cm, 32pt);
		\draw [darkgray, fill] (9.89cm, 32pt) circle [radius=0.04];
		
		% Labels		
		\draw (0,0) node[below right=3pt, align=left] {\textbf{1977}\\Sanger with\\chain-terminating\\inhibitors};
		\draw (3.87,0) node[above left=3pt, align=right] {\textbf{1986}\\First semi-automated\\DNA sequencing machine};
		\draw (4.30,0) node[below right=3pt, align=left] {\textbf{1987}\\First fully automated\\DNA sequencing machine};
		\draw (8.17,0) node[above left=3pt, align=right] {\textbf{1996}\\Born of\\pyrosequencing technique};
		\draw (9.03,0) node[below right=3pt, align=left] {\textbf{1998}\\Publication\\of ``Phred''\\software};
		\draw (9.89,0) node[above left=3pt, align=right] {\textbf{2000}\\First\\NGS\\machine};
				
		% Horizontal timeline			
		\draw  [darkgray, thick] (0,0) -- (9.89,0);
		\draw  [->] [darkgray, dashed, thick] (9.89,0) -- (10.7,0);
	\end{tikzpicture}	
	}
	\end{center}
	\tiny{\hfill{\cite{sanger1977dna, smith1986fluorescence, prober1987system, ronaghi1996real, ewing1998base, brenner2000gene}}}
	\end{block}
	%
	\pause
	\scriptsize{%
	\only<2>{%
	\begin{columns}
		\begin{column}{0.6\textwidth}\\
			\tiny{%
			\begin{tikzpicture}
				\draw (0,5) node[inner sep=0]{\includegraphics[width=\textwidth]{./images/seq_cost}};
				\draw (0.2,5) node[above=1.6cm, align=center]{\scriptsize{Average cost for raw megabase of DNA sequence}};
				
				% Pyrosequencing
			    \draw (-1.4,3.52) -- (-1.4,4.5);
			    \draw [fill] (-1.4,4.5) circle [radius=0.03];
			    \draw (-1.4,4.5) node[right=1pt, align=left] {454\\pyrosequencing};
			    
    				% Illumina
			    \draw (-1.1,3.52) -- (-1.1,4);
			    \draw [fill] (-1.1,4) circle [radius=0.03];
			    \draw (-1.1,4) node[right=1pt, align=left] {Illumina};
			    
     			% SOLiD
			    \draw (0.2,3.52) -- (0.2,4);
			    \draw [fill] (0.2,4) circle [radius=0.03];
			    \draw (0.2,4) node[right=1pt, align=left] {SOLiD};
			    
        			% Ion Torrent
			    \draw (1.3,3.52) -- (1.3,5);
			    \draw [fill] (1.3,5) circle [radius=0.03];
			    \draw (1.3,5) node[left=1pt, align=right] {Ion\\Torrent};
			    
			 	% PacBio
			    \draw (2.2,3.52) -- (2.2,4.3);
			    \draw [fill] (2.2,4.3) circle [radius=0.03];
			    \draw (2.2,4.3) node[left=1pt, align=right] {PacBio};			    
			\end{tikzpicture}
			}%
		\end{column}
		%
		\begin{column}{0.4\textwidth}
			\normalsize{%
			With the development of new technologies the cost for sequencing one megabase of DNA has been reduced by more than 1000 times
			}
		\end{column}		
	\end{columns}	
	}
	\only<3>{%
	\begin{columns}
		\begin{column}{0.6\textwidth}
			\begin{center}
				\begin{tikzpicture}
				\draw (0,5.5) node[inner sep=0]{\includegraphics[width=0.9\textwidth]{./images/seq_number_genebank}};
				\draw (0.2,5.5) node[above=1.5cm, align=center]{Number of genome sequences submitted to Genbank};
				
				% NGS era
				\draw (1,4.25) -- (1,6);
				\draw [fill] (1,6) circle [radius=0.03];
				\draw (1,6) node[above=1pt, align=center] {NGS era};
				\end{tikzpicture}	
			\end{center}
		\end{column}
		%
		\begin{column}{0.4\textwidth}
			\normalsize{%
			The ``NGS era'' has led to a drastic increment of genome sequences available in public databases
			}	
		\end{column}		
	\end{columns}
	}
	}
	\end{overlayarea}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 5
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 6
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\tikzstyle{era} = [text=black, rectangle, draw, fill=blue!20, text width=1.8cm, text centered, rounded corners, minimum height=0.8cm]
	\tikzstyle{bad} = [text=black, rectangle, draw, fill=red!20, text width=4cm, text centered, rounded corners, minimum height=1cm]
	\tikzstyle{good} = [text=black, rectangle, draw, fill=green!20, text width=4cm, text centered, rounded corners, minimum height=1cm]
	\tikzstyle{normal} = [black, text=black, rectangle, draw, fill=white, rounded corners]
	\tikzstyle{problems} = [black, text=black, rectangle, draw, fill=black!10, rounded corners]	
	\tikzstyle{line} = [draw, -latex']
	
\begin{overlayarea}{\textwidth}{0.9\textheight}	
\begin{tikzpicture}[node distance = 0.3cm, auto]
	\scriptsize{%
	\only<1->{%
    % Place nodes
    \node [era] (NGS) {\normalsize{\textbf{NGS Era}}};
    \node [good, below right=of NGS, align=left] (disciplines) {%
    		\parbox{4cm}{%
			\begin{itemize}
			    \item Born of Metagenomics
			    \item Improvements in Genomics 
			\end{itemize}%
		}
	};
    \node [bad, below left=of NGS, align=left] (problems) {%
	    \parbox{4cm}{%
			\begin{itemize}
			    \item Huge amount of data
			    \item Less human control 
			\end{itemize}%
		}
    	};    	
    	% Draw edges
    \path [line] (NGS) to [out=0, in=90] (disciplines);
    \path [line] (NGS) to [out=180, in=90] (problems);
    	}
    	\only<2->{%
	% Other two nodes    	
    	\node [problems, below=1cm of problems, align=left, node distance=2cm] (metprob){%
    		\textbf{Technical Problems:}\\
    		\parbox{4cm}{%
			\begin{itemize}
			    \item[*] Handling huge data			    
			    \item[*] Mining databases
			    \item[*] Control of generated data
			    \item[*] Lack of dedicated tools
			\end{itemize}%
		}
    	};
    	
    	\node [problems, below right=1cm and 1cm of problems, align=left, node distance=2cm] (bioprob){%
    		\textbf{Biological Problems:}\\
    		\parbox{4cm}{%
			\begin{itemize}
			    \item[*] Difficult interpretation			    
			    \item[*] Hard visualization
			    \item[*] Selection of significant data
			    \item[*] Lack of standard methods
			\end{itemize}%
		}
    	};
    
    % Other edges
    \path [line] (problems) to [out=270, in=90] (metprob);
    \path [line] (problems) to [out=275, in=90] (bioprob);
    }
    
    \only<3->{%
    	% Other two nodes    	
    	\node [normal, below=0.8cm of metprob, align=left, node distance=2cm] (bigtool){%
    		\textbf{Small tool for Big data:}\\
    		\parbox{4cm}{% 	
		\begin{itemize}
			\item[\checkmark] Raw data control
			\item[\checkmark] Tools evaluation
		\end{itemize}
    		}
    	};
    	
    	\node [normal, right=0.6cm of bigtool, align=left, node distance=2cm] (bactenv){%
    		\parbox[t]{2.8cm}{%
    			\textbf{Bacterial world:}\\
    			\vspace{-2.5mm}
    			\begin{itemize}
    				\item[\checkmark] Bacteria in the environment    				
    			\end{itemize}
		}
		\parbox[t]{2.8cm}{%
			\textbf{Bacterial landlord:}\\
			\vspace{-2.5mm}
			\begin{itemize}
				\item[\checkmark] Bacterial symbiosis
				\item[\checkmark] Bacterial infections
			\end{itemize}
		}
    	};
    
    % Other edges
    \path [line] (metprob) to [out=270, in=90] (bigtool);
    \path [line] (bioprob) to [out=270, in=90] (bactenv);
    }
    }
\end{tikzpicture}
\end{overlayarea}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 6
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - BACKGROUND
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - SMALL TOOLS FOR BIG DATA
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Small tools for big data}
\subsection{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 7
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\textbf{\Large{DNA sequencing control}}
	\begin{columns}
		\begin{column}{0.7\textwidth}
			Chromatogram:
			\includegraphics[width=\textwidth]{./images/dna_sequence.png}		
		\end{column}
		\begin{column}{0.3\textwidth}
			Quality:
			\includegraphics[width=\textwidth]{./images/per_base_quality_before}
		\end{column}		
	\end{columns}
	\begin{columns}
		\begin{column}{0.35\textwidth}
			\begin{align*}
				Q &= Quality\\
				P &= Error Probability\\
				\\
				Q &= -10 \, \log_{10} P\\
				P &= 10^{-^{Q}/_{10}}\\
			\end{align*}		
		\end{column}
		\begin{column}{0.65\textwidth}
			\only<1>{%
			\begin{block}{}
				A quality value $Q$ is an integer representing the probability that the corresponding base call is incorrect ($P$).\\
				Wrong bases may lead to the production of incorrect data in downstream analyses.
			\end{block}
			}
			\only<2>{%
				\includegraphics[width=0.8\textwidth]{./images/Dont_panic}
			}
		\end{column}		
	\end{columns}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 7
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 8
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\vspace{2mm}
	\begin{overlayarea}{\textwidth}{0.9\textheight}
	\begin{tikzpicture}
		\node [inner sep=0] (logo) {\includegraphics[width=0.56\textwidth]{./images/streamingtrim}};
		\node [right=0.6cm of logo.west, anchor=south west] (text) {\textbf{\Large{StreamingTrim 1.0}}};
		\node [below=0.7cm of logo.south west, anchor=north west,align=left] (pipeline) {Dynamic Window Algorithm\\\includegraphics[width=0.5\textwidth]{./images/dynamic_pipeline_all}};
		\only<1>{%
			\node [right=0.5cm of pipeline, align=right] (explained) {\parbox{4cm}{%
				The algorithm is able to retain as much biological information as possible using small amount of memory.
			}};
		}
		\only<2->{%			
			\node [right=0.2cm of logo.north east, anchor=north west] (assigned) {\includegraphics[width=0.39\textwidth]{./images/assigned}};
			\node [inner sep=0, below=0.3cm of assigned] (zscore) {\includegraphics[width=0.43\textwidth]{./images/zscore}};
			\node [draw=black, fill=white, below=0.3cm of zscore.south east, anchor=north east, rounded corners] (frm) {\tiny{%
					$Z_{score} = \log_{10} \bigg( \frac{Quality\:improvement}{Length\:decrease} \bigg)$}};
		}
	\end{tikzpicture}
	\end{overlayarea}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 8
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 9
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\begin{overlayarea}{\textwidth}{0.99\textheight}
		\begin{tikzpicture}
			\only<1>{%
			\node [anchor=south west] (descript) at (0,0) {
				\begin{minipage}{0.97\textwidth}%
 				\begin{block}{RDP Classifier}%
					\small{\textit{The RDP Classifier is a naive Bayesian classifier which was developed to provide rapid taxonomic placement based on rRNA sequence data. The RDP Classifier can rapidly and accurately classify bacterial and archaeal 16s rRNA sequences, and Fungal LSU sequences. It provides taxonomic assignments from domain to genus, with confidence estimates for each assignment.}\\
				\hfill{[RDP staff]}}
				\end{block}
			 	\end{minipage}};						 
			}
				
			\node [inner sep=0, anchor=north west] at (0,-0.3) (rdpperc) {\includegraphics[width=0.4\textwidth]{./images/rdp_percentages}};
			
			\only<1>{%
			\node [right=0.5cm of rdpperc] (textone) {\parbox{5.7cm}{
				These results were obtained using simulated datasets \cite{wang2007naive}; but what about real ones?
			}};	
			}
			
			\only<2->{%
			\node [anchor=south west, inner sep=0.37cm] (genrdp) {\includegraphics[width=0.45\textwidth]{./images/genus_rdp}};
			\node [right=0.7cm of rdpperc.south east, anchor= south west] (allrdp) {\includegraphics[width=0.51\textwidth]{./images/all_rdp}};
			\node [above=0.2cm of allrdp.north east, anchor=south east] (text) {\parbox{5.3cm}{
				Real datasets display a lower rate of assignements than simulated ones at all taxonomic levels.
			}}
			};	  
		\end{tikzpicture}
	\end{overlayarea}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 9
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 10
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\textbf{\Large{Scientific Production:}}
	\begin{itemize}
		\item \textbf{Bacci, G.}, Bazzicalupo, M., Benedetti, A., \& Mengoni, A. (2014). StreamingTrim 1.0: a Java software for dynamic trimming of 16S rRNA sequence data from metagenetic studies. Molecular ecology resources, 14(2), 426-434.
		\item \textbf{Bacci, G.}, Bani, A., Bazzicalupo, M., Ceccherini, M.T., Galardini, M., Nannipieri, P., Pietramellara, G., \& Mengoni, A. (2015). Evaluation of the performances of Ribosomal Database Project (RDP) Classifier for taxonomic assignment of 16S rRNA metabarcoding sequences generated from Illumina-Solexa NGS. \textit{Journal of Genomics}, 3, 36-39.
	\end{itemize}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 10
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - SMALL TOOLS FOR BIG DATA
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - BACTERIAL WORLD
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Bacterial World}
\subsection{}

\newif\iffondimod
\fondimodtrue

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 11 - REMOVED
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begin{frame}
%	\textbf{\Large{Bacteria-environment interaction}}
%	\begin{block}{}
%		Determining the effects of agricultural management, along with seasonal variations, on soil bacterial community through DGGE and soil biochemical properties.
%	\end{block}
%	\begin{center}
%		\includegraphics[width=0.5\textwidth]{./images/soilsink_env}	
%	\end{center}
%	\begin{columns}
%		\begin{column}{0.5\textwidth}
%			It is possible to define two clusters for each season, except for the cork-oak forest (CO) which forms two separated clusters.
%		\end{column}
%		\begin{column}{0.5\textwidth}
%			\begin{tikzpicture}
%				\node<1-> [inner sep=0, anchor=south west] (heatmap) at (0,0) {\includegraphics[width=\textwidth]{./images/heatmap_soilsink}};
%				
%				\begin{scope}[x={(heatmap.south east)},y={(heatmap.north west)}]
%				\draw<2> [yellow, thick] (0.02, 0.571) rectangle (0.71, 0.88);
%				\draw<2> [green, thick] (0.02,0.38) rectangle (0.71, 0.569);
%				\draw<3> [blue, thick] (0.02,0.26) rectangle (0.71,0.38);
%				\end{scope}
%			\end{tikzpicture}
%		\end{column}
%	\end{columns}
%\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 11
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 12
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\iffondimod
\begin{frame}
	\vspace{3mm}
	\begin{overlayarea}{\textwidth}{0.95\textheight}
	\textbf{\Large{The pan-bacteriome model}}
	\begin{columns}
		\begin{column}{0.5\textwidth}
			\begin{center}
				\only<1>{\includegraphics[width=\textwidth]{./images/bacteriome/bacteriome_1}}
				\only<2>{\includegraphics[width=\textwidth]{./images/bacteriome/bacteriome_2}}
				\only<3->{\includegraphics[width=\textwidth]{./images/bacteriome/bacteriome_3}}				
			\end{center}
		\end{column}
		\begin{column}{0.6\textwidth}
			\begin{itemize}
				\item<1-3> Considering all bacterial strains living in a particular environment
				\item<2-3> It is possible to classify them in different samples
				\item<3> And define a core and accessory bacteriome according to their presence/absence patterns in the collected samples
			\end{itemize}
		\end{column}
	\end{columns}
	\vspace{5mm}
	\only<4->{%
	\begin{columns}
		\begin{column}{0.64\textwidth}
			\begin{block}{}
			\scriptsize{%
				 Inspecting the dynamics able to shape bacterial communities in three soil mesocosms exposed to different environmental conditions through time using the 16S rRNA massive sequencing in combination with the pan-bacteriome approach.
			}
			\end{block}
		\end{column}
		\begin{column}{0.36\textwidth}
			\vskip -55mm
			\includegraphics[width=\textwidth]{./images/pipeline_panbact}
		\end{column}		
	\end{columns}	
	}
	\end{overlayarea}
\end{frame}
\else
\begin{frame}
	\vspace{3mm}
	\begin{overlayarea}{\textwidth}{0.95\textheight}
	\textbf{\Large{The pan-bacteriome model}}
	\begin{columns}
		\begin{column}{0.5\textwidth}
			\begin{center}
				\only<1>{\includegraphics[width=\textwidth]{./images/bacteriome/bacteriome_1}}
				\only<2>{\includegraphics[width=\textwidth]{./images/bacteriome/bacteriome_2}}
				\only<3->{\includegraphics[width=\textwidth]{./images/bacteriome/bacteriome_3}}				
			\end{center}
		\end{column}
		\begin{column}{0.6\textwidth}
			\begin{itemize}
				\item<1-> Considering all bacterial strains living in a particular environment
				\item<2-> It is possible to classify them in different samples
				\item<3-> And define a core and accessory bacteriome according to their presence/absence patterns in the collected samples
			\end{itemize}
		\end{column}
	\end{columns}
	\vspace{5mm}
	\only<4->{%
	\begin{block}{}
		 Inspecting the dynamics able to shape bacterial communities in three soil mesocosms exposed to different environmental conditions through time using the 16S rRNA massive sequencing in combination with the pan-bacteriome approach.
	\end{block}
	}
	\end{overlayarea}
\end{frame}
\fi

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 12
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 13
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\tikzstyle{line} = [draw, -latex']
	\begin{overlayarea}{\textwidth}{0.99\textheight}
	\vspace{2mm}
	\begin{tikzpicture}
		\node<1-> [inner sep=0, anchor=north west] (heatmap) at (0,0) {\includegraphics[width=0.5\textwidth]{./images/bacteriome/Fig3}};
		\node<1-> [inner sep=0, above, anchor=south west] (hetlab) at (heatmap.north west) {Bacterial community abundance};
		
		\node<2-> [inner sep=0, right=1cm of heatmap.east] (venn) {\includegraphics[width=0.28\textwidth]{./images/bacteriome/Fig4}};
		\node<2-> [inner sep=0, above, anchor=south west] (vennlab) at (venn.north west) {Pseudo-Venn diagram};
		\path<2-4> [line, thick] (heatmap) -- (venn);
		
		\node<3-> [inner sep=0.2cm, below=0.3cm of heatmap.south west, anchor=north west] (heatmaptwo) {\includegraphics[width=0.4\textwidth]
{./images/bacteriome/Fig5}};
		\path<3-4> [line, thick] (venn) to [out=190, in=45] (heatmaptwo);
		\node<3-> [inner sep=0, below] (htwolab) at (heatmaptwo.south) {Comparing communities};
		
		\node<4-> [inner sep=0.5, right=0.6cm of heatmaptwo.south east, anchor=south west] (composition) {\includegraphics[width=0.43\textwidth]
{./images/bacteriome/Fig6}};
		\path<4> [line, thick] (heatmaptwo) -- (composition);
		\node<4-> [inner sep=0, below] (complab) at (composition.south) {Genera composition};		
		
		\draw<5-> [white, fill=white, fill opacity=.8] (0.01,-3.0) rectangle (10.1,0.31);
		\node<5-> [inner sep=0,above=1cm of composition.north, anchor=south east] (textblock) {\begin{minipage}{6cm}
			\begin{block}{}
				Core and accessory bacteriomes have two distinct bacterial community in both {\color{red} high} and {\color{blue} low} classification levels.
			\end{block}
		\end{minipage}
		};
		
		%\draw[help lines] (0,0) grid (11,-8);
		\draw<5-> [red, thick] (0.2,-7) rectangle (4.7,-3.5);
		\draw<5-> [blue, thick] (5.2,-7) rectangle (10.1,-3.5);
		
		\path<5-> [line, red, thick] (textblock) to [out=195, in=90] (heatmaptwo);
		\path<5-> [line, blue, thick] (textblock) to [out=212, in=90] (composition);		
	\end{tikzpicture}
	\end{overlayarea}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 13
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START- Slide 14
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\textbf{\Large{Analysis of bacterial communities}}
	\vspace{5mm}
	
	\begin{itemize}
		\item Low resolution approach using DGGE combined with soil biochemical properties
		\begin{enumerate}
			\item Agricultural management and seasonal variations in a Mediterranean region (Sardinia)
		\end{enumerate}
		\item High resolution approach through 16S rRNA massive sequencing
		\begin{enumerate}
			\item Rhizosphere and bulk soil of a saline pond in Inner Mongolia (China)
			\item A natural saline soil located in Sicily (Italy)
			\item Along the beach gradient in three beaches of Favignana island (Italy)		
		\end{enumerate}		
	\end{itemize}
	\vspace{3mm}
	\begin{block}{Aims}
		Inspect differences in bacterial community composition in respect with environment characteristics like: salinity, beach gradient and rhizosphere effects.
	\end{block}
\end{frame}

%%%%%%%%%% MODIFIED %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begin{frame}
%	\textbf{\Large{Analysis of bacterial communities}}
%	\vspace{5mm}
%	\begin{itemize}
%		\item Rhizosphere and bulk soil of a saline pond in Inner Mongolia (China)
%		\item A natural saline soil located in Sicily (Italy)
%		\item Along the beach gradient in three beaches of Favignana island (Italy)
%	\end{itemize}
%	\vspace{3mm}
%	\begin{block}{Aims}
%		Inspect differences in bacterial community composition in respect with environment characteristics like: salinity, entropy gradient and rhizosphere effects. In all studies the 16S rRNA massive sequencing has been applyed to deeply investigate changes in the composition of bacterial communities.
%	\end{block}
%\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 14
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 15
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\iffondimod
\begin{frame}
	\vspace{8mm}
	\begin{overlayarea}{\textwidth}{0.80\textheight}
	\begin{tikzpicture}
		\scriptsize{%
		\node [anchor=south east] (txtsseek) at (0,0) {\begin{minipage}{0.45\textwidth}
			\begin{block}{\scriptsize{Menagement Vs Seasons}}
				It is possible to define two clusters for each season, except for the cork-oak forest (CO) which forms two separated clusters.
			\end{block}
		\end{minipage}};
		\node [inner sep=0, right=0.3cm of txtsseek.north east, anchor=north west] (sseek) {\includegraphics[width=0.47\textwidth]{./images/heatmap_soilsink}};
		\begin{scope}[shift=(sseek.south west), x={(sseek.south east)},y={(sseek.north west)}]
			\draw [yellow, thick] (0.02, 0.571) rectangle (0.71, 0.88);
			\draw [green, thick] (0.02,0.38) rectangle (0.71, 0.569);
			\draw [blue, thick] (0.02,0.26) rectangle (0.71,0.38);
		\end{scope}
				
		\node [inner sep=0, below=1.2cm of txtsseek.south west, anchor=north west] (rhizo) {\includegraphics[width=0.5\textwidth]{./images/bulk_rhizo}};
		\node [right=0.3cm of rhizo] (txtrhizo) {\begin{minipage}{0.45\textwidth}
			\begin{block}{\scriptsize{Rhizosphere Vs Bulk}}
				Bulk sediment samples grouped separately from rhizosphere samples, which clustered in distinctive couples.
			\end{block}
		\end{minipage}};
		\begin{scope}[shift=(rhizo.south west), x={(rhizo.south east)},y={(rhizo.north west)}]
			\draw [blue, thick, rotate around={-6:(0.2,0.57)}] (0.2,0.57) ellipse (0.35cm and 1.2cm);
			\draw [green, thick, rotate around={8:(0.91,0.55)}] (0.91,0.55) ellipse (0.35cm and 0.2cm);
		\end{scope}
		}
	\end{tikzpicture}	
	\end{overlayarea}
\end{frame}

\begin{frame}
	\vspace{6mm}
	\begin{overlayarea}{\textwidth}{0.80\textheight}
		\begin{tikzpicture}
			\scriptsize{%
				\node [inner sep=0] (salinity) {\includegraphics[width=0.45\textwidth]{./images/salinity}};
				\node [left=0.3cm of salinity] (txtsalinity) {\begin{minipage}{0.45\textwidth}
				\begin{block}{\scriptsize{Salinity effect}}
					High salinity samples and low salinity samples showed two distinct bacterial distributions.
				\end{block}
				\end{minipage}};
				\begin{scope}[shift=(salinity.south west), x={(salinity.south east)}, y={(salinity.north west)}]
					\draw [red,->, thick] (0.0, 0.3) -- (0.0, 0.9);
				\end{scope}
				
				\node [inner sep=0, below=0.8cm of txtsalinity] (favignana) {\includegraphics[width=0.45\textwidth]{./images/favignana}};
				\node [right=0.3cm of favignana] (txtfavignana) {\begin{minipage}{0.45\textwidth}
				\begin{block}{\scriptsize{Beach gradient}}
					It is possible to detect some abundance trends in bacterial Phyla following the beach gradient.
				\end{block}
				\end{minipage}};
				\begin{scope}[shift=(favignana.south west), x={(favignana.south east)}, y={(favignana.north west)}]
					\draw [green,->, thick] (0.05, 0.0) -- (0.85, 0.0);
				\end{scope}
			}
		\end{tikzpicture}
	\end{overlayarea}
\end{frame}
\else
\begin{frame}
	\vspace{8mm}
	\begin{overlayarea}{\textwidth}{0.80\textheight}
	\begin{tikzpicture}
		\scriptsize{%
		\node<1-> [anchor=south east] (txtsseek) at (0,0) {\begin{minipage}{0.45\textwidth}
			\begin{block}{\scriptsize{Menagement Vs Seasons}}
				It is possible to define two clusters for each season, except for the cork-oak forest (CO) which forms two separated clusters.
			\end{block}
		\end{minipage}};
		\node<1-> [inner sep=0, right=0.3cm of txtsseek.north east, anchor=north west] (sseek) {\includegraphics[width=0.47\textwidth]{./images/heatmap_soilsink}};
		\begin{scope}[shift=(sseek.south west), x={(sseek.south east)},y={(sseek.north west)}]
			\draw<2> [yellow, thick] (0.02, 0.571) rectangle (0.71, 0.88);
			\draw<2> [green, thick] (0.02,0.38) rectangle (0.71, 0.569);
			\draw<3-> [blue, thick] (0.02,0.26) rectangle (0.71,0.38);
		\end{scope}
				
		\node<4-> [inner sep=0, below=1.2cm of txtsseek.south west, anchor=north west] (rhizo) {\includegraphics[width=0.5\textwidth]{./images/bulk_rhizo}};
		\node<4-> [right=0.3cm of rhizo] (txtrhizo) {\begin{minipage}{0.45\textwidth}
			\begin{block}{\scriptsize{Rhizosphere Vs Bulk}}
				Bulk sediment samples grouped separately from rhizosphere samples, which clustered in distinctive couples.
			\end{block}
		\end{minipage}};
		\only<5>{
		\begin{scope}[shift=(rhizo.south west), x={(rhizo.south east)},y={(rhizo.north west)}]
			\draw [blue, thick, rotate around={-6:(0.2,0.57)}] (0.2,0.57) ellipse (0.35cm and 1.2cm);
			\draw [green, thick, rotate around={8:(0.91,0.55)}] (0.91,0.55) ellipse (0.35cm and 0.2cm);
		\end{scope}
		}
		}
	\end{tikzpicture}	
	\end{overlayarea}
\end{frame}

\begin{frame}
	\vspace{6mm}
	\begin{overlayarea}{\textwidth}{0.80\textheight}
		\begin{tikzpicture}
			\scriptsize{%
				\node<1-> [inner sep=0] (salinity) {\includegraphics[width=0.45\textwidth]{./images/salinity}};
				\node<1-> [left=0.3cm of salinity] (txtsalinity) {\begin{minipage}{0.45\textwidth}
				\begin{block}{\scriptsize{Salinity effect}}
					High salinity samples and low salinity samples showed two distinct bacterial distributions.
				\end{block}
				\end{minipage}};
				\begin{scope}[shift=(salinity.south west), x={(salinity.south east)}, y={(salinity.north west)}]
					\draw<2-> [red,->, thick] (0.0, 0.3) -- (0.0, 0.9);
				\end{scope}
				
				\node<3-> [inner sep=0, below=0.8cm of txtsalinity] (favignana) {\includegraphics[width=0.45\textwidth]{./images/favignana}};
				\node<3-> [right=0.3cm of favignana] (txtfavignana) {\begin{minipage}{0.45\textwidth}
				\begin{block}{\scriptsize{Beach gradient}}
					It is possible to detect some abundance trends in bacterial Phyla following the beach gradient.
				\end{block}
				\end{minipage}};
				\only<4->{%
				\begin{scope}[shift=(favignana.south west), x={(favignana.south east)}, y={(favignana.north west)}]
					\draw [green,->, thick] (0.05, 0.0) -- (0.85, 0.0);
				\end{scope}
				}
			}
		\end{tikzpicture}
	\end{overlayarea}
\end{frame}
\fi

%%%%%%%%%%%%%%%%%%% MODIFIED %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\begin{frame}
%	\vspace{2mm}
%	\begin{overlayarea}{\textwidth}{0.97\textheight}
%	\begin{tikzpicture}
%		\scriptsize{%
%		\node<1-> [inner sep=0] (rhizo) {\includegraphics[width=0.5\textwidth]{./images/bulk_rhizo}};
%		\node<1-> [right=0.3cm of rhizo] (txtrhizo) {\begin{minipage}{0.45\textwidth}
%			\begin{block}{\scriptsize{Rhizosphere Vs Bulk}}
%				Bulk sediment samples grouped separately from rhizosphere samples, which clustered in distinctive couples.
%			\end{block}
%		\end{minipage}};
%		
%		\node<2-> [inner sep=0, below=0.2cm of txtrhizo] (salinity) {\includegraphics[width=0.45\textwidth]{./images/salinity}};
%		\node<2-> [left=0.3cm of salinity] (txtsalinity) {\begin{minipage}{0.45\textwidth}
%			\begin{block}{\scriptsize{Salinity effect}}
%				High salinity samples and low salinity samples showed two distinct bacterial distributions.
%			\end{block}
%		\end{minipage}};
%		
%		\node<3-> [inner sep=0, below=0.2cm of txtsalinity] (favignana) {\includegraphics[width=0.45\textwidth]{./images/favignana}};
%		\node<3-> [right=0.3cm of favignana] (txtfavignana) {\begin{minipage}{0.45\textwidth}
%			\begin{block}{\scriptsize{Beach gradient}}
%				It is possible to detect some abundance trends in bacterial Phyla following the beach gradient.
%			\end{block}
%		\end{minipage}};
%		}
%	\end{tikzpicture}	
%	\end{overlayarea}
%\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 15
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 16
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\textbf{\Large{Scientific Production:}}
	\footnotesize{%
	\begin{itemize}
		\item Bevivino, A., Paganin, P., \textbf{Bacci, G.}, Florio, A., Pellicer, M. S., Papaleo, M. C., ... \& Dalmastri, C. (2014). Soil Bacterial Community Response to Differences in Agricultural Management along with Seasonal Changes in a Mediterranean Region. PloS one, 9(8), e105515.
		\item \textbf{Bacci, G.}, Ceccherini, M. T., Bani, A., Bazzicalupo, M., Castaldini, M., Galardini, M., ... \& Mengoni, A. (2015). Exploring the dynamics of bacterial community composition in soil: the pan-bacteriome approach. Antonie van Leeuwenhoek, 1-13.
		\item Borruso, L., \textbf{Bacci, G.}, Mengoni, A., De Philippis, R., \& Brusetti, L. (2014). Rhizosphere effect and salinity competing to shape microbial communities in Phragmites australis (Cav.) Trin. ex-Steud. FEMS microbiology letters, 359(2), 193-200.
		\item Canfora, L., \textbf{Bacci, G.}, Pinzari, F., Papa, G. L., Dazzi, C., \& Benedetti, A. (2014). Salinity and bacterial diversity: to what extent does the concentration of salt affect the bacterial community in a saline soil?. PloS one, 9(9), e106662.
		\item \textbf{Bacci, G.}, Pagoto, E., Passaponti, M., Vannocci, P., Ugolini, A., \& Mengoni, A. (2014). Composition of supralittoral sediments bacterial communities in a Mediterranean island. Annals of Microbiology, 1-13.
	\end{itemize}
	}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 16
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - BACTERIAL WORLD
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - BACTERIAL LANDLORD
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Bacterial landlord}
\subsection{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 17
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\textbf{\Large{Analysis of microbiomes}}
	\vspace{4mm}
	\begin{itemize}
		\item Low resolution approach using T-RFLP profiling
		\begin{enumerate}
			\item Analysis of bacterial communities associated with \textit{Talitrus saltator}
			\item Analysis of lung bacterial communities in Cystic Fibrosis patients
		\end{enumerate}
		\item High resolution approach using 16S rRNA massive sequencing
		\begin{enumerate}
			\item Analysis of gut microbiota of talitrid amphipods
			\item Analysis of the airway microbiota of Cystic Fibrosis patients
		\end{enumerate}
	\end{itemize}
	\vspace{4mm}
	\begin{block}{Aims}
		Find a connection between the microbiota composition and different conditions of the hosts.
	\end{block}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 17
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 18
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\iffondimod
\begin{frame}
	\vspace{8mm}
	\begin{overlayarea}{\textwidth}{0.90\textheight}
	\begin{tikzpicture}
		\scriptsize{%
		\node [anchor=south east] (pcatxt) at (0,0) {\begin{minipage}{0.4\textwidth}
			\begin{block}{\scriptsize{Conditions Vs TRFs}}
				Differential occurrence of some putative species like \textit{P. aeruginosa} and \textit{S. aureus}
			\end{block}
		\end{minipage}};
		\node [inner sep=0, right=0.3cm of pcatxt.north east, anchor=north west] (pca) {\includegraphics[width=0.55\textwidth]{./images/pca_both}};
		\begin{scope}[shift=(pca.south west), x={(pca.south east)}, y={(pca.north west)}]
			%\draw [help lines,step=0.05] (0.0,0.0) grid (1.0,1.0);	
			\draw [red] (0.1,0.76) ellipse (0.3cm and 0.1cm);
			\draw [red] (0.09,0.47) ellipse (0.3cm and 0.1cm);
			
			\draw [red] (0.6,0.48) ellipse (0.3cm and 0.1cm);
			\draw [red] (0.6,0.88) ellipse (0.3cm and 0.1cm);
		\end{scope}
		
		\node [inner sep=0, below=1.2cm of pcatxt.south west, anchor=north west] (nmdstal) {\includegraphics[width=0.47\textwidth]{./images/nmds_talitri}};
		\node [right=0.3cm of nmdstal] (txtnmdstal) {\begin{minipage}{0.45\textwidth}
			\begin{block}{\scriptsize{Different sampling sites}}
				No strict clustering was observed based on different sampling sites
			\end{block}
		\end{minipage}};
		}
	\end{tikzpicture}	
	\end{overlayarea}
\end{frame}
\else
\begin{frame}
	\vspace{8mm}
	\begin{overlayarea}{\textwidth}{0.90\textheight}
	\begin{tikzpicture}
		\scriptsize{%
		\node<1-> [anchor=south east] (pcatxt) at (0,0) {\begin{minipage}{0.4\textwidth}
			\begin{block}{\scriptsize{Conditions Vs TRFs}}
				Differential occurrence of some putative species like \textit{P. aeruginosa} and \textit{S. aureus}
			\end{block}
		\end{minipage}};
		\node<1-> [inner sep=0, right=0.3cm of pcatxt.north east, anchor=north west] (pca) {\includegraphics[width=0.55\textwidth]{./images/pca_both}};
		\only<2->{%
		\begin{scope}[shift=(pca.south west), x={(pca.south east)}, y={(pca.north west)}]
			%\draw [help lines,step=0.05] (0.0,0.0) grid (1.0,1.0);	
			\draw [red] (0.1,0.76) ellipse (0.3cm and 0.1cm);
			\draw [red] (0.09,0.47) ellipse (0.3cm and 0.1cm);
			
			\draw [red] (0.6,0.48) ellipse (0.3cm and 0.1cm);
			\draw [red] (0.6,0.88) ellipse (0.3cm and 0.1cm);
		\end{scope}
		}
		\node<3-> [inner sep=0, below=1.2cm of pcatxt.south west, anchor=north west] (nmdstal) {\includegraphics[width=0.47\textwidth]{./images/nmds_talitri}};
		\node<3-> [right=0.3cm of nmdstal] (txtnmdstal) {\begin{minipage}{0.45\textwidth}
			\begin{block}{\scriptsize{Different sampling sites}}
				No strict clustering was observed based on different sampling sites
			\end{block}
		\end{minipage}};
		}
	\end{tikzpicture}	
	\end{overlayarea}
\end{frame}
\fi
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 18
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 19
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\vspace{2mm}
	\textbf{\Large{Correlation network approach}}
	\vspace{1mm}
	\pause
	\begin{columns}
		\begin{column}{0.5\textwidth}
			\begin{overlayarea}{\textwidth}{0.8\textheight}
				\begin{center}
				\only<2>{\includegraphics[width=0.8\textwidth]{./images/network_table_1}\\
				\vspace{4mm}}
				
				\only<3>{\includegraphics[width=0.8\textwidth]{./images/network_table_1}\\
				\vspace{4mm}
				\includegraphics[width=0.8\textwidth]{./images/network_nodes_1}}	
				
				\only<4>{\includegraphics[width=0.8\textwidth]{./images/network_table_2}\\
				\vspace{4mm}
				\includegraphics[width=0.8\textwidth]{./images/network_nodes_1}}
				
				\only<5->{\includegraphics[width=0.8\textwidth]{./images/network_table_2}\\
				\vspace{4mm}
				\includegraphics[width=0.8\textwidth]{./images/network_nodes_2}}															
				\end{center}
			\end{overlayarea}
		\end{column}
		\begin{column}{0.5\textwidth}
			\begin{overlayarea}{\textwidth}{0.8\textheight}
				\begin{itemize}
					\vspace{8mm}
					\item<2-> In a dataset with 3 samples and 3 detected genera
					\item<3-> It is possible to represent each genus as a node
					\item<4-> If two genera show a positive correlation (their abundance patterns are similar)
					\item<5-> It is possible to put a vertex (link) between the nodes representing the correlated genera	
				\end{itemize}
			\end{overlayarea}
		\end{column}
	\end{columns}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 19
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 20
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\iffondimod
\begin{frame}
	\textbf{\Large{Talitrid amphipods microbiome}}
	\vspace{5mm}
	\begin{columns}
		\begin{column}{0.6\textwidth}
			\includegraphics[width=\textwidth]{./images/talitri_maps}
		\end{column}
		\begin{column}{0.4\textwidth}
			\tikzstyle{pipetal} = [black, rectangle, draw, rounded corners, text width=3.5cm, text centered, font=\scriptsize]
			\tikzstyle{line} = [draw, -latex']
			\begin{tikzpicture}[node distance=1.4cm, auto]
				\node[pipetal] (nodeone) {Collecting 12 pools of talitrid amphipods in 7 sites (5 different species)};
				\node[pipetal, below of=nodeone] (nodetwo) {Etracting bacterial DNA from gut};
				\node[pipetal, below of=nodetwo] (nodethree) {Selection and amplification of the V4 region of 16S rRNA gene through PCR};
				\node[pipetal, below of=nodethree] (nodefour) {Sequencing of produced fragments};
				\node[pipetal, below of=nodefour] (nodefive) {Clustering and Taxonomic annotation};
				
				\path[line] (nodeone) -- (nodetwo);
				\path[line] (nodetwo) -- (nodethree);
				\path[line] (nodethree) -- (nodefour);
				\path[line] (nodefour) -- (nodefive);
			\end{tikzpicture}
%			\begin{block}{}
%				Inspecting the gut microbiome of 5 species of talitrid amphipods in 7 distinct sites through Correlation Network analysis.			
%			\end{block}
		\end{column}
	\end{columns}
\end{frame}
\fi
\begin{frame}
	\tikzstyle{stepnode} = [black, text=black, rectangle, draw, fill=white, rounded corners, text width=4cm, align=center]
	\tikzstyle{line} = [draw, -latex']
	\vspace{3mm}
	\begin{overlayarea}{\textwidth}{0.95\textheight}	
	\begin{tikzpicture}
		\node [inner sep=0] (talnetwork) {\includegraphics[width=0.5\textwidth]{./images/networks/Figure_3_talkaled}}; 
		\scriptsize{%
		\node<1-> [stepnode, right=0.8cm of talnetwork.north east, anchor=north west] (stepone) {Generating correlation network comprehending  all \textit{Talitrus} species};
		\node<2-> [stepnode, below=1cm of stepone] (steptwo) {Finding the most informative number of clusters using ``Intra-Cluster Clustering Coefficient'' (ICCC)};
		\path<2-> [line] (stepone) -- (steptwo);
		\node<3-> [stepnode, below=1cm of steptwo] (stepthree) {Generating clusters through ``Markov Cluster Algorithm'' (MCL) \cite{van2001graph}};
		\path<3-> [line] (steptwo) -- (stepthree);
		}
		\node<4-> [inner sep=0, below=0.5cm of stepthree] (nets) {\includegraphics[width=0.45\textwidth]{./images/networks/Suppl_Mat_Figure_S1}};
	\end{tikzpicture}
	\end{overlayarea}	
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 20
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 21
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\iffondimod
\begin{frame}
	\vspace{2mm}
	\textbf{\Large{Airway microbiome of CF patients}}
	\small{%
	\begin{columns}
		\begin{column}{0.5\textwidth}
			\begin{center}
				\includegraphics[width=0.8\textwidth]{./images/pipeline_mod_1}
			\end{center}
		\end{column}
		\begin{column}{0.5\textwidth}
			\tikzstyle{pipefc} = [black, rectangle, draw, rounded corners, text width=3.5cm, text centered, font=\scriptsize]
			\tikzstyle{line} = [draw, -latex']
			\begin{tikzpicture}[node distance=1.4cm, auto]
				\node[pipefc] (nodeone) {52 Patients enrolled and divided into 2 main groups and 3 sub groups based on their conditions};
				\node[pipefc, below of=nodeone] (nodetwo) {Collecting sputum samples from each patient};
				\node[pipefc, below of=nodetwo] (nodethree) {Selection and amplification of the V3, V4, and V5 regions of 16S rRNA gene through PCR};
				\node[pipefc, below of=nodethree] (nodefour) {Sequencing of produced fragments};
				\node[pipefc, below of=nodefour] (nodefive) {Clustering and Taxonomic annotation};
				
				\path[line] (nodeone) -- (nodetwo);
				\path[line] (nodetwo) -- (nodethree);
				\path[line] (nodethree) -- (nodefour);
				\path[line] (nodefour) -- (nodefive);
			\end{tikzpicture}
%			\begin{enumerate}
%				\item Patients were divided into 2 main groups and 3 sub-groups based on their rate decline in FEV\textsubscript{1}\% value per year and their FEV\textsubscript{1}\% values, respectively.
%				\item Bacterial DNA was extracted from sputum sample of all patients. 
%				\item 16S rRNA gene was amplified and sequenced using the Roche 454 GS FLX+ System (158107 sequences were generated with a mean length of 400bp).
%				\item Generated sequences were clustered into ``Operational Taxonomic Units'' (OTUs) with UPARSE choosing a single representative sequence for each OTU.
%				\item Representative sequences were taxonomically classified using SINA classifier.
%			\end{enumerate}
		\end{column}
	\end{columns}
	}
\end{frame}
\fi

\begin{frame}
	\begin{overlayarea}{\textwidth}{0.9\textheight}	
	\begin{tikzpicture}
		\scriptsize{%
		\node<1-> [inner sep=0, anchor=south west] (abprev) {\includegraphics[width=0.5\textwidth]{./images/FC/Figure_2}};
		\node<2-> [right=0.3cm of abprev.north east, anchor=north west] (txtabprev) {\begin{minipage}{0.4\textwidth}
			\begin{block}{}
				Despite the high variability it is possible to detect an almost ``{\color{green} constant}'' part of the community and a {\color{red} scattered} one.
			\end{block}
		\end{minipage}};
		\only<2->{%
		\begin{scope}[shift=(abprev.south west), x={(abprev.south east)}, y={(abprev.north west)}]
			\draw [green, thick] (0.0,0.275) rectangle (0.8,0.88);
			\draw [red, thick] (0.0,0.07) rectangle (0.8,0.27);
		\end{scope}
		}
		\node<3-> [inner sep=0, below=0.1cm of txtabprev] (cca) {\includegraphics[width=0.38\textwidth]{./images/FC/Figure_4}};
		\node<3-> [below=0.0cm of cca] (txtcca) {\begin{minipage}{0.4\textwidth}
			\begin{block}{}
				CCA analysis on OTU composition has shown that patients have different bacterial communities depending on their status.
			\end{block}
		\end{minipage}};
		}
	\end{tikzpicture}
	\end{overlayarea}	
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 22
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 23
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\tikzstyle{line} = [draw, -latex']
	\begin{overlayarea}{\textwidth}{0.9\textheight}
	\begin{tikzpicture}
		\scriptsize{%
		\node<1-> [inner sep=0] (network) {\includegraphics[width=0.6\textwidth]{./images/FC/Figure_5}};	
		\node<2-> [right=0.2cm of network.north east, anchor=north west] (txtnetwork) {\begin{minipage}{0.35\textwidth}
			\begin{block}{}
				With the worsening of the disease patients show a less interconnected network with a lower number of nodes and a lower degree.
			\end{block}
		\end{minipage}};
		
		\node<3-> [inner sep=0, below=0.2cm of network] (model) {\includegraphics[width=0.4\textwidth]{./images/FC/Figure_6_16s}};
		\path<3-> [line, thick] (network) to [out=350, in=0] (model);
		
		\node<3-> [below=3cm of txtnetwork] (txtmodel) {\begin{minipage}{0.35\textwidth}
			\begin{block}{}
				A relatively complex community is present only in patients with low level of disease, then something happens paving the way for the lung colonization of pathological strains.
			\end{block}
		\end{minipage}};
		}
	\end{tikzpicture}
	\end{overlayarea}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 23
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 24
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\textbf{\Large{Scientific Production:}}
	\footnotesize{%
	\begin{itemize}
		\item Mengoni, A., Focardi, A., \textbf{Bacci, G.}, \& Ugolini, A. (2013). High genetic diversity and variability of bacterial communities associated with the sandhopper \textit{Talitrus saltator} (Montagu)(Crustacea, Amphipoda). \textit{Estuarine, Coastal and Shelf Science}, 131, 75-82.
		\item \textbf{Bacci, G.}, Abdelrhman, K.F.A., Marras, B., Nistri, A., Schintu, M., Ugolini, A., \& Mengoni, A. The gut microbiota of talitrid amphipods provides insight into the ecology of supralittoral sediments detritivors. Manuscripts submitted to: \textit{FEMS Microbiology Ecology}.
		\item Paganin, P., Fiscarelli, E.V., Tuccio, V., Chiancianesi, M., \textbf{Bacci, G.}, Morelli, P., Dolce, D., Dalmastri, C., De Alessandri, A., Lucidi, V., Taccetti, G., Mengoni, A., \& Bevivino, A. Changes in Cystic Fibrosis Airway Microbial Community Associated with a Severe Decline in Lung Function. Manuscripts submitted to: \textit{PloS one}.
		\item \textbf{Bacci, G.}, Paganin, P., Lopez, L., Vanni, C., Dalmastri, C., Cantale, C., Daddiego, L., Perrotta, G., Taccetti, G., De Alessandri, A., Fiscarelli, E.E., Lucidi, V., Bevivino, A., \& Mengoni, A. Taxonomic signatures of CF airway microbiota distinguish between stable and severe declining patients. Manuscripts to be  submitted to: \textit{Proceedings of the National Academy of Sciences}
	\end{itemize}
	}
\end{frame}	
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 24
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
	
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - BACTERIAL LANDLORD
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - CONCLUSIONS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Conclusions}
\subsection{}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - Slide 25
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}
	\vspace{2mm}
	\textbf{\Large{Conclusions and future perspective}}
	\vspace{2mm}	
	\small{%
	\begin{itemize}
		\item Metagenomics has emerged as a new discipline of biology fuelled by substantial technological advancements in DNA sequencing and analysis of produced data.
		\item All these technological and conceptual advances have introduced new problems related to the analysis of produced data.
%		\item  The huge number of sequences generated in metagenomic studies is often difficult to handle with existing tools, not specifically designed for this purpose (e.g. the available database searching algorithms are very slow if used for metagenomic data).
		\item   Developing statistical methods and software explicitly designed to cope with these problems is becoming mandatory for making a step forward in this novel discipline.
	\end{itemize}
	\pause
	\begin{block}{}
		While the technical aspects of metagenomics are now mainly devoted to company services which provide sequencing services, the bioinformatic skill is pivotal for data interpretation and study design.
	\end{block}		
	}
\end{frame}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - Slide 25
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% END - CONCLUSIONS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% START - END STUFF
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{}
\begin{frame}
	\vspace{2mm}
	{\huge\textbf{Thanks to:}}
	\vspace{3mm}
	\begin{columns}[T]
		\begin{column}{0.5\textwidth}
			{\large\textbf{Dep. of Biology UniFi}}\\
			Alessio Mengoni\\
			Marco Bazzicalupo\\
			Renato Fani\\
			Alberto Ugolini\\
			Alice Checcucci\\
			Khaled Abdelrhman\\ 
			Isabel Maida\\			
			Francesco Vitali	\\
			Elena Perrin\\
			\vspace{2mm}
			{\large\textbf{UniBz}}\\
			Luigimaria Borruso\\
			Lorenzo Brusetti\\			
		\end{column}
		\begin{column}{0.5\textwidth}
			{\large\textbf{CRA-RPS}}\\
			Anna Benedetti\\
			\vspace{2mm}
			{\large\textbf{COMBO Group}}\\
			Marco Fondi\\
			Marco Galardini\\
			Emanuele Bosi\\
			Luana Presta\\				
			\vspace{2mm}
			{\large\textbf{ENEA}}\\
			Annamaria Bevivino\\
			\vspace{2mm}
			{\large\textbf{DISPAA}}\\			
			Paolo Nannipieri\\
			Maria Teresa Ceccherini\\
		\end{column}
	\end{columns}
	\vspace{8mm}
	\let\thefootnote\relax\large{... and all people I forgot to mention!}
\end{frame}

\begin{frame}
	\begin{center}
		{\huge\textbf{Thank you for your attention!}}
		\includegraphics[width=0.55\textwidth]{./images/computerneeded.png}	
	\end{center}
	\let\thefootnote\relax\small{\textbf{Contacts:}\\ Mail: giovanni.bacci@unifi.it \\ Twitter: @combogenomics}
\end{frame}

\begin{frame}[allowframebreaks, t]
	\vspace{2mm}
	\textbf{\Large{References}}
	\vspace{2mm}	
	\tiny
	\bibliography{References}
\end{frame}

\end{document}